dockers/docker-database/docker-database-init.sh

```diff
@@ -41,14 +41,16 @@ then
     fi
 fi
 
+export BMP_DB_PORT=6400
```

Comment:
>6400 [](http://example.com/codeflow?start=19&length=4)

@Pterosaur How to ensure this port is not conflicting with REMOTE_DB_PORT? or does it matter?

Reply:
Yes, it's possible. But according to the current smart-switch architecture, the 19 port quotas are sufficient.